sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -320,6 +321,10 @@ highlightFunction
     : HIGHLIGHT LR_BRACKET relevanceField (COMMA highlightArg)* RR_BRACKET
     ;
 
+matchQueryAltSyntaxFunction
```

Comment:
just in case we want to add more functions here, we should call this something more genetic.  Such as `equalFieldRelevanceFunction` or something similar?

Reply:
Or `altSingleFieldRelevanceFunction` and move under `relevanceFunction`.  That'll make it slightly easier to refactor the syntax down the line.

Reply:
Fixed in 1073f074d944305159b6b9ec52a7b81cd75fbad7